pkg/workloads/tf_train/train_util.py

```diff
@@ -33,6 +33,24 @@ def get_input_placeholder(model_name, ctx, training=True):
     return input_placeholder
 
 
+def get_label_placeholder(model_name, ctx):
+    model = ctx.models[model_name]
+
+    target_column_name = model["target_column"]
+    column_type = tf_lib.CORTEX_TYPE_TO_TF_TYPE[ctx.columns[target_column_name]["type"]]
+    return tf.placeholder(shape=[None], dtype=column_type)
+
+
+def get_transform_tensor_fn(ctx, model_impl, model_name):
+    model = ctx.models[model_name]
+    model_config = ctx.model_config(model["name"])
+
+    def transform_tensor_fn_wrapper(inputs, labels):
+        return model_impl.transform_tensorflow(inputs, labels, model_config)
```

Comment:
Can you validate the method signature of the user's `transform_tensorflow()` implementation? I think you might want to add it to `get_model_impl()` in `context.py` since it's similar to how we validate `create_estimator()`, but I would check with @vishalbollu

Reply:
Agree